README.md

```diff
@@ -34,46 +34,118 @@ Simmy offers the following chaos-injection policies:
 
 |Policy| What does the policy do?|
 | ------------- |------------- |
-|**[Fault](#Inject-fault)**|Injects exceptions or substitute results, to fake faults in your system.|
+|**[Exception](#Inject-exception)**|Injects exceptions in your system.|
+|**[Result](#Inject-result)**|Substitute results to fake faults in your system.|
 |**[Latency](#Inject-latency)**|Injects latency into executions before the calls are made.|
 |**[Behavior](#Inject-behavior)**|Allows you to inject _any_ extra behaviour, before a call is placed. |
 
 # Usage
 
```

Comment:
Should we give a full example here, before we give the syntax for the specific parts/cases?  (the syntax for parts might make more sense after the reader has seen a full example)

    // Following example causes the policy to throw SocketException with a probability of 5% if enabled
    var fault = new SocketException(errorCode: 10013);
    var chaosPolicy = MonkeyPolicy.InjectException(with =>
        with.Fault(fault)
            .InjectionRate(0.05)
            .Enabled()
        );